php/Terminus/Environment.php

```diff
@@ -301,6 +301,54 @@ public function wake() {
   }
 
   /**
+   * Merge the Master Branch (Dev Environment) into this Multidev Branch (Environment)
+   *
+   */
+  public function mergeFromMaster($options = array()) {
+    if (!$this->isMultidev()) {
+      throw new Exception(sprintf("The %s environment is not a multidev environment", $this->id));
+    }
+
+    $default_params = array(
+      'updatedb' => false
+    );
+    $params = array_merge($default_params, $options);
+
+    $workflow = $this->site->workflows->create('merge_dev_into_cloud_development_environment', array(
+      'environment' => $this->id,
+      'params' => $params
+    ));
+
+    return $workflow;
+  }
+
+  /**
+   * Merge this Multidev Branch (Environment) into the Master Branch (Dev Environment)
+   *
+   */
+  public function mergeToMaster($options = array()) {
+    if (!$this->isMultidev()) {
+      throw new Exception(sprintf("The %s environment is not a multidev environment", $this->id));
+    }
+
+    $default_params = array(
+      'updatedb' => false
+    );
+    $params = array_merge($default_params, $options);
+
+    // This function is a little odd because we invoke it on a
+    // multidev environment, but it applies a workflow to the 'dev' environment
+    $params['from_environment'] = $this->id;
```

Comment:
If you take and put 
`'from_environment` => $this->id`
in the options array from the calling function, you can eliminate this whole function.